rosflight/src/rosflight_io.cpp

```diff
@@ -616,25 +616,39 @@ void rosflightIO::handle_small_mag_msg(const mavlink_message_t &msg)
   mag_pub_.publish(mag_msg);
 }
 
-void rosflightIO::handle_small_sonar(const mavlink_message_t &msg)
+void rosflightIO::handle_small_range(const mavlink_message_t &msg)
 {
-  mavlink_small_sonar_t sonar;
-  mavlink_msg_small_sonar_decode(&msg, &sonar);
+  mavlink_small_range_t mRange;
```

Comment:
Just as a style convention thing, could you rename the `mRange` variable to `range`? This is in keeping with the [ROS C++ style guide](http://wiki.ros.org/CppStyleGuide)